tests/unit/test_entries.py

```diff
@@ -503,6 +503,20 @@ def test_to_api_repr_defaults(self):
         }
         self.assertEqual(entry.to_api_repr(), expected)
 
+    def test_to_api_repr_struct(self):
```

Comment:
please, no abbreviations in test or other method names

Reply:
"to_api_repr" is an existing function that we can't change here without breaking the API. We could add this to the list for v3.0.0 changes (but it may not be worth the user confusion)